onchange_helper/README.rst

```diff
@@ -0,0 +1,47 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===============
+Onchange Helper
+===============
+
+This is a technical module. Its goal is to ease the play of onchange method directly called from python file.
```

Comment:
I just add a quick example to make it clearer.
Here is a real one  : 
https://github.com/akretion/account-financial-tools/blob/9-account_asset_management/account_asset_management/models/account_move.py#L115